### ISSUE TITLE
Convertir función `get_indice_cuatrimestral()` a una versión general respecto al período (que pase a ser un parámetro)

### ISSUE THREAD

**Issue by pablotis** [dnme-minturdep/herramientas#3]
original: 

`get_indice_cuatrimestral <- function(x) (round(x / stats::lag(x, n = 4)-1, digits = 3) * 100)`

**Comment by jurricariet** [comment]
Acá metí PR para ver si está OK. Habría que chequear que esté OK! https://github.com/dnme-minturdep/herramientas/pull/7

**Comment by pablotis** [comment]
Ahí estuve viéndolo!

Los chequeos los podés probar en el paquete mismo, en la carpeta `test/testthat/` podés agregar los test con la función `usethis::use_test("nombre_de_la_funcion")` (te crea un script propio para esta función) y un modelo de cómo sería crear el testeo, para chequear que la función de los resultados que esperás:

Ejemplo:
```
library(testhat)

test_that("multiplication works", {
  
  var1 <- c(10,10,10,15)
  
  variacion <- tasa_variacion(variable = var1, 
                 periodo = "anual", 
                 porcentaje = TRUE)
  
  
  expect_equal(variacion[4], 50)

  })
```

Más allá de esto, me queda la duda si no es mejor dejar el parámetro `periodo` en numérico, pensando en que la función sirva para distintas bases y la cantidad de posiciones (que entiendo  que es como se calcula la variación) no esté atada a una referencia en texto (una posición puede ser un trimestre, un año o un mes 🤔, dependiendo con qué tipo de base trabajemos)

**Comment by jurricariet** [comment]
Genial lo del testeo.
Con respecto al parámetro `periodo`, se puede poner el número que quieras para determinar la cantidad de posiciones, luego están las opciones predeterminadas (anual = 1, mensual = 12, trimestral = 4), al menos esa era la idea!

**Comment by pablotis** [comment]
Perfecto! Paso el PR. Podríamos agregarle condiciones para asegurarnos de que sí o sí se pongan los valores numéricos de `1:12` o en texto `"anual | trimestral | mensual"`

**Comment by TuQmano** [comment]
Este PR ya está mergeado como para cerrar, no? @jurricariet @pablotis

**Comment by jurricariet** [comment]
Sí! Estaba pensando si las opciones predefinidas del parámetro `periodo` no pueden confundir un poco. Por ejemplo, `periodo = "mensual"` quiere decir que calcula con respecto a 12 posiciones para atrás. Entonces es la tasa interanual para una serie con frecuencia mensual. Todas las predefinidas están referidas a variaciones interanuales y lo que se especifica es la frecuencia de la serie, tal vez aclarando eso se entienda.

**Comment by TuQmano** [comment]
Esto en que estado está? @jurricariet @pablotis
